src/api/checkout/checkout_payment_activations/v1/_swagger.json.tpl

```diff
@@ -94,6 +100,12 @@
               "$ref": "#/definitions/ProblemJson"
             }
           },
+          "424": {
+            "description": "Invalid `Nodo` response",
```

Comment:
We could improve the description message, think about it 😁

Reply:
Maybe "Unexpected error. The requests cannot be handled by PagoPa services." sounds better?